packages/client/wallets/smart-wallet/src/error/index.ts

```diff
@@ -40,6 +44,28 @@ export class CrossmintServiceError extends SmartWalletSDKError {
     }
 }
 
+export class AdminMismatchError extends SmartWalletSDKError {
+    public readonly required: SignerDisplay;
+    public readonly used?: SignerDisplay;
```

Comment:
This `SignerDisplay` type differs from the spec, but it's better since it includes the passkey public key.